Top/csound.c

```diff
@@ -1767,10 +1741,12 @@ int kperf_nodebug(CSOUND *csound)
     /* for one kcnt: */
     if (csound->oparms_.sfread)         /*   if audio_infile open  */
       csound->spinrecv(csound);         /*      fill the spin buf  */
-    csound->spoutactive = 0;            /*   make spout inactive   */
     /* clear spout */
     memset(csound->spout, 0, csound->nspout*sizeof(MYFLT));
-    memset(csound->spraw, 0, csound->nspout*sizeof(MYFLT));
+    memset(csound->spraw,0,
+           sizeof(MYFLT)*csound->nspout*csound->oparms->numThreads);
+    // 27.1.24 spraw is not used anymore
```

Comment:
Seems like this comment is a bit confusing about spraw not used anymore? Looks like it is still in use. What do you think about giving spraw a different name, or at least adding comments to the field in csoundCore.h? I suspect future devs would appreciate some documentation about the spout vs spraw.

Reply:
I'll think of a name change. I removed it earlier but then restored it.